experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/Controllers/BotController.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+
+namespace LivePersonAgentBot.Controllers
+{
+    // This ASP Controller is created to handle a request. Dependency Injection will provide the Adapter and IBot
+    // implementation at runtime. Multiple different IBot implementations running at different endpoints can be
+    // achieved by specifying a more specific type for the bot constructor argument.
+    [Route("api/messages")]
+    [ApiController]
+    public class BotController : ControllerBase
+    {
+        private readonly IBotFrameworkHttpAdapter Adapter;
```

Comment:
our current coding standards for this are "_adapter". We're starting to enforce that via the Analyzers in the main repos, but if it's easy I would update this code as well.

Reply:
I'll address this consistently in all projects in the new iteration -- more changes coming later.